k8s/production/custom/rebuild-everything/sealed-secrets.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: bitnami.com/v1alpha1
+kind: SealedSecret
+metadata:
+  name: rebuild-everything
+  namespace: custom
+spec:
+  encryptedData:
+    # This token must have repo:status permissions
```

Comment:
Is this comment accurate?  Isn't this supposed to be an ssh key?

Reply:
ssh key registered with the gitlab API with that access level

Reply:
I stole the ssh stuff from the `gh-gl-sync`. I am basically doing the same thing, but I only want to do one tiny part and I also don't want to add more to that script if I don't need to.